.github/workflows/release.yml

```diff
@@ -40,7 +40,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN || github.token }}
         uses: changesets/action@v1
         with:
-          version: node scripts/version-alpha.mjs
           title: Version Packages (v3)
 
       - name: Check if release needs to be published
```

Comment:
This check is `hardhat` package specific. Also, the next steps build and publish all the packages under the `next` tag which needs to be updated, too.